assets/js/blocks/active-filters/active-attribute-filters.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * External dependencies
+ */
+import { useCollection, useQueryStateByKey } from '@woocommerce/base-hooks';
+
+/**
+ * Internal dependencies
+ */
+import { renderItem } from './utils';
+import { removeAttributeFilterBySlug } from '../../utils/attributes-query';
+
+/**
+ * Component that renders active attribute (terms) filters.
+ */
+const ActiveAttributeFilters = ( { attributeObject = {}, slugs = [] } ) => {
+	const { results, isLoading } = useCollection( {
+		namespace: '/wc/store',
+		resourceName: 'products/attributes/terms',
+		resourceValues: [ attributeObject.id ],
+	} );
+
+	const [ productAttributes, setProductAttributes ] = useQueryStateByKey(
+		'product-grid',
+		'attributes',
+		[]
+	);
+
+	if ( isLoading ) {
+		return null;
+	}
+
+	const attributeLabel = attributeObject.label;
+
+	return slugs.map( ( slug ) => {
+		const termObject =
+			results.find( ( term ) => {
+				return term.slug === slug;
+			} ) || null;
```

Comment:
If `find` returns no results `termObject` will be `null`, but in the next line we are reading `termObject.name`, which will throw an error if `termObject` is `null`.

If we are sure `find` will always return a value, you can remove `|| null` from here, otherwise we should add a check that `termObject` is an object.